### PR TITLE
Specify details of buildTarget/run

### PR DESCRIPTION
**Pull request by agluszak**

### CODE REVIEW

**Comment by agluszak** [comment]
Hmm, I don't know where the whitespace changes are coming from

File: website/generated/docs/specification.md [hunk 1]
```diff
@@ -1029,23 +1077,34 @@ The run request is sent from the client to the server to run a build target. The
 server communicates during the initialize handshake whether this method is
 supported or not.
 
+Note that an empty run request is valid. Run will be executed in the target as
```
Comment (adpi2) [inline website/generated/docs/specification.md:1080]: It's unclear what a empty run request is. Is it a run request with a null `params`?

File: website/generated/docs/extensions/jvm.md [hunk 2]
```diff
@@ -30,6 +29,7 @@ after all the targets are compiled.
 
 #### JvmTestEnvironmentParams
 
+
```
Comment (adpi2) [inline website/generated/docs/extensions/jvm.md:32]: Would be good to understand where do the empty space comes from and revert them?

File: website/generated/docs/specification.md (hunk 1 above)
Comment (adpi2) [reply website/generated/docs/specification.md:1080]: Oh now I see there is a related discussion: https://github.com/build-server-protocol/build-server-protocol/discussions/636

**Review by adpi2 — APPROVED** [review]
Perfect, thanks

**Pull request merged**